app/assets/javascripts/common/reports.js

```diff
@@ -421,37 +435,34 @@ DashboardReports = () => {
                 return value + "%";
               },
             },
-          },
-          tooltips: {
-            mode: "x",
-            intersect: false,
-            displayColors: false,
-            xAlign: "center",
-            yAlign: "top",
-            xPadding: 6,
-            yPadding: 6,
-            caretSize: 3,
-            caretPadding: 1,
-            callbacks: {
-              title: function () {
-                return "";
-              },
-              label: function (tooltipItem, data) {
-                let numerators = Object.values(
-                    data.datasets[tooltipItem.datasetIndex].numerators
-                );
-                let denominators = Object.values(
-                    data.datasets[tooltipItem.datasetIndex].denominators
-                );
-                return (
-                    formatNumberWithCommas(numerators[tooltipItem.index]) +
-                    " of " +
-                    formatNumberWithCommas(denominators[tooltipItem.index]) +
-                    " follow-up patients"
-                );
+            tooltip: {
+              mode: "x", // not required
+              intersect: false, // not required
+              enabled: true, // not required?
```

Comment:
Let's finish this cleanup before shipping

Reply:
I have left this here for the time being because:
- I am unsure if this is required in the current place, it needs some research to check its not needed - here I am flagging for a check
- I am half way through the next PR where I am removing unused config and want to not mix the update with a new DRY-ing.
I separated these steps to update, then DRY.
This PR was purely to get v3.9.1 working and to keep design consistency.